src/types/dict.rs

```diff
@@ -780,7 +780,7 @@ where
 }
 
 /// Represents a tuple which can be used as a PyDict item.
-trait PyDictItem<'py> {
+pub trait PyDictItem<'py> {
```

Comment:
This looks a possibly unintentional change

```suggestion
trait PyDictItem<'py> {
```

Reply:
Removed! For some reason I thought I needed this for one of the tests.